src/gui/advancedsettings.cpp

```diff
@@ -559,7 +566,13 @@ void AdvancedSettings::loadAdvancedSettings()
     // Announce IP
     m_lineEditAnnounceIP.setText(session->announceIP());
     addRow(ANNOUNCE_IP, tr("IP Address to report to trackers (requires restart)"), &m_lineEditAnnounceIP);
-    // stop tracker timeout
+#if (LIBTORRENT_VERSION_NUM >= 10207)
+    // Max concurrent HTTP announces
+    m_spinBoxMaxConcurrentHTTPAnnounces.setValue(session->maxConcurrentHTTPAnnounces());
+    addRow(MAX_CONCURRENT_HTTP_ANNOUNCES, (tr("Max Concurrent HTTP Announces") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#max_concurrent_http_announces", "(?)"))
```

Comment:
Please fix this text to follow the style of everything else (and in WebUI too):
`Max concurrent HTTP announces`